package.json

```diff
@@ -4,7 +4,11 @@
   "description": "A dynamic app to create objects",
   "main": "index.js",
   "scripts": {
-    "test": "npm test"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build:dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack serve --mode development --open",
+    "mode": "development"
   },
```

Comment:
Some of your run commands don't perform as expected e.g. when I run the `npm start` command, I noticed that the `webpack-dev-server` needed to be installed since it wasn't part of your dependencies and after the installation, the app wasn't still displayed. Please try to test your commands and ensure that they run without errors.